images/eh-expansion-evaluator/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG VERSION=${VERSION:-eb982ae41f83aaf44d4a8142980f7a41ac33e979}
+
+# Install Git to clone the repository
+RUN apt-get update && \
+    apt-get install -y \
+    git \
+    && \
+    apt-get clean -y
+
+# Clone the repository
+RUN git clone https://github.com/oscarlr/EH-expansion-evaluator
```

And checkout the desired version (for repeatability if we ever need to rebuild the image):

```suggestion
RUN git clone https://github.com/oscarlr/EH-expansion-evaluator && cd EH-expansion-evaluator  && git checkout $VERSION
```